alpine image armhf / openhab not starting

Hi,

there seems to be something with the alpine image for armhf systems. It seems like alpine boots, than openhab fires up but crashes. After a few minutes the docker daemon restarts the image. 
Interesting fact is that in the first iteration my syslog spits out kernel warnings from the host machine.

Tested on:

Cubietruck:
- Debian Stretch (Installation from scratch via Debian Installer)
- Kernel: 4.9.0-6-armmp-lpae
- Docker: 18.04.0~ce~3-0~debian

Raspberry Pi 3:
- Fresh install of Raspbian Stretch
- Kernel: 4.14.34-v7+
- Docker version 18.05.0-ce, build f150324

It happens on both systems. 

My Compose File:
```
version: '2.1'

services:
  openhab:
    image: "openhab/openhab:2.2.0-armhf-alpine"
    restart: always
    container_name: oh-alpine-arp
    network_mode: host
    tty: true
    logging:
      driver: syslog
    volumes:
      - "/etc/localtime:/etc/localtime:ro"
      - "/etc/timezone:/etc/timezone:ro"
      - "/docker/volumes/openhab-alpine/addons:/openhab/addons"
      - "/docker/volumes/openhab-alpine/conf:/openhab/conf"
      - "/docker/volumes/openhab-alpine/userdata:/openhab/userdata"
    command: "./start_debug.sh"
```

My syslog with enabled syslog driver. 1st Iteration with kernel warnings:
```
May 19 11:28:47 raspberrypi dockerd[5425]: time="2018-05-19T11:28:47+02:00" level=info msg="shim docker-containerd-shim started" address="/containerd-shim/moby/e404b082aef5c4621883de2814ebc927d2221ec47236dd4023095c621bccdc74/shim.sock" debug=false module="containerd/tasks" pid=5868
May 19 11:28:48 raspberrypi dockerd[5425]: time="2018-05-19T11:28:48+02:00" level=warning msg="OOM monitoring failed" error="cgroups: memory cgroup not supported on this system"
May 19 11:28:48 raspberrypi e404b082aef5[5425]: + test -t 0
May 19 11:28:48 raspberrypi e404b082aef5[5425]: + [ 0 -eq 1 ]
May 19 11:28:48 raspberrypi e404b082aef5[5425]: + set -euo pipefail
May 19 11:28:48 raspberrypi e404b082aef5[5425]: + IFS=
May 19 11:28:48 raspberrypi e404b082aef5[5425]:
May 19 11:28:48 raspberrypi e404b082aef5[5425]: + [ limited = unlimited ]
May 19 11:28:48 raspberrypi e404b082aef5[5425]: + rm -f /openhab/runtime/instances/instance.properties
May 19 11:28:48 raspberrypi e404b082aef5[5425]: + rm -f /openhab/userdata/tmp/instances/instance.properties
May 19 11:28:48 raspberrypi e404b082aef5[5425]: + NEW_USER_ID=9001
May 19 11:28:48 raspberrypi e404b082aef5[5425]: + echo Starting with openhab user id: 9001
May 19 11:28:48 raspberrypi e404b082aef5[5425]: Starting with openhab user id: 9001
May 19 11:28:48 raspberrypi e404b082aef5[5425]: + id -u openhab
May 19 11:28:48 raspberrypi e404b082aef5[5425]: + echo Create user openhab with id 9001
May 19 11:28:48 raspberrypi e404b082aef5[5425]: Create user openhab with id 9001
May 19 11:28:48 raspberrypi e404b082aef5[5425]: + adduser -u 9001 -D -g  -h /openhab openhab
May 19 11:28:48 raspberrypi e404b082aef5[5425]: + ls -A /openhab/userdata
May 19 11:28:48 raspberrypi e404b082aef5[5425]: + [ -z  ]
May 19 11:28:48 raspberrypi e404b082aef5[5425]: + echo No userdata found... initializing.
May 19 11:28:48 raspberrypi e404b082aef5[5425]: No userdata found... initializing.
May 19 11:28:48 raspberrypi e404b082aef5[5425]: + cp -av /openhab/userdata.dist/. /openhab/userdata/
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./logs/openhab.log' -> '/openhab/userdata/./logs/openhab.log'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./logs' -> '/openhab/userdata/./logs'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./tmp/README' -> '/openhab/userdata/./tmp/README'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./tmp' -> '/openhab/userdata/./tmp'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/org.jupnp.cfg' -> '/openhab/userdata/./etc/org.jupnp.cfg'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/org.apache.karaf.command.acl.scope_bundle.cfg' -> '/openhab/userdata/./etc/org.apache.karaf.command.acl.scope_bundle.cfg'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/org.apache.karaf.features.repos.cfg' -> '/openhab/userdata/./etc/org.apache.karaf.features.repos.cfg'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/system.properties' -> '/openhab/userdata/./etc/system.properties'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/org.apache.karaf.command.acl.system.cfg' -> '/openhab/userdata/./etc/org.apache.karaf.command.acl.system.cfg'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/org.eclipse.smarthome.voice.cfg' -> '/openhab/userdata/./etc/org.eclipse.smarthome.voice.cfg'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/org.apache.karaf.log.cfg' -> '/openhab/userdata/./etc/org.apache.karaf.log.cfg'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/org.apache.felix.fileinstall-deploy.cfg' -> '/openhab/userdata/./etc/org.apache.felix.fileinstall-deploy.cfg'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/distribution.info' -> '/openhab/userdata/./etc/distribution.info'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/keystore' -> '/openhab/userdata/./etc/keystore'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/com.eclipsesource.jaxrs.swagger.config.cfg' -> '/openhab/userdata/./etc/com.eclipsesource.jaxrs.swagger.config.cfg'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/org.ops4j.pax.logging.cfg' -> '/openhab/userdata/./etc/org.ops4j.pax.logging.cfg'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/org.apache.karaf.command.acl.config.cfg' -> '/openhab/userdata/./etc/org.apache.karaf.command.acl.config.cfg'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/org.apache.karaf.command.acl.jaas.cfg' -> '/openhab/userdata/./etc/org.apache.karaf.command.acl.jaas.cfg'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/branding-ssh.properties' -> '/openhab/userdata/./etc/branding-ssh.properties'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/custom.properties' -> '/openhab/userdata/./etc/custom.properties'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/scripts/shell.completion.script' -> '/openhab/userdata/./etc/scripts/shell.completion.script'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/scripts' -> '/openhab/userdata/./etc/scripts'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/org.apache.karaf.command.acl.kar.cfg' -> '/openhab/userdata/./etc/org.apache.karaf.command.acl.kar.cfg'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/equinox-debug.properties' -> '/openhab/userdata/./etc/equinox-debug.properties'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/org.apache.karaf.kar.cfg' -> '/openhab/userdata/./etc/org.apache.karaf.kar.cfg'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/custom.system.properties' -> '/openhab/userdata/./etc/custom.system.properties'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/org.apache.karaf.jaas.cfg' -> '/openhab/userdata/./etc/org.apache.karaf.jaas.cfg'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/org.apache.felix.eventadmin.impl.EventAdmin.cfg' -> '/openhab/userdata/./etc/org.apache.felix.eventadmin.impl.EventAdmin.cfg'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/profile.cfg' -> '/openhab/userdata/./etc/profile.cfg'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/version.properties' -> '/openhab/userdata/./etc/version.properties'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/org.apache.karaf.command.acl.shell.cfg' -> '/openhab/userdata/./etc/org.apache.karaf.command.acl.shell.cfg'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/config.properties' -> '/openhab/userdata/./etc/config.properties'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/org.apache.karaf.features.cfg' -> '/openhab/userdata/./etc/org.apache.karaf.features.cfg'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/keys.properties' -> '/openhab/userdata/./etc/keys.properties'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/org.apache.karaf.command.acl.feature.cfg' -> '/openhab/userdata/./etc/org.apache.karaf.command.acl.feature.cfg'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/jmx.acl.org.apache.karaf.config.cfg' -> '/openhab/userdata/./etc/jmx.acl.org.apache.karaf.config.cfg'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/com.eclipsesource.jaxrs.connector.cfg' -> '/openhab/userdata/./etc/com.eclipsesource.jaxrs.connector.cfg'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/overrides.properties' -> '/openhab/userdata/./etc/overrides.properties'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/startup.properties' -> '/openhab/userdata/./etc/startup.properties'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/org.eclipse.smarthome.audio.cfg' -> '/openhab/userdata/./etc/org.eclipse.smarthome.audio.cfg'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/users.properties' -> '/openhab/userdata/./etc/users.properties'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/jmx.acl.org.apache.karaf.bundle.cfg' -> '/openhab/userdata/./etc/jmx.acl.org.apache.karaf.bundle.cfg'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/java.util.logging.properties' -> '/openhab/userdata/./etc/java.util.logging.properties'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/all.policy' -> '/openhab/userdata/./etc/all.policy'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/org.ops4j.pax.url.mvn.cfg' -> '/openhab/userdata/./etc/org.ops4j.pax.url.mvn.cfg'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/org.ops4j.pax.web.cfg' -> '/openhab/userdata/./etc/org.ops4j.pax.web.cfg'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/jre.properties' -> '/openhab/userdata/./etc/jre.properties'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/org.apache.karaf.command.acl.bundle.cfg' -> '/openhab/userdata/./etc/org.apache.karaf.command.acl.bundle.cfg'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/branding.properties' -> '/openhab/userdata/./etc/branding.properties'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/shell.init.script' -> '/openhab/userdata/./etc/shell.init.script'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc/org.apache.karaf.shell.cfg' -> '/openhab/userdata/./etc/org.apache.karaf.shell.cfg'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/./etc' -> '/openhab/userdata/./etc'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/userdata.dist/.' -> '/openhab/userdata/.'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: + cmp /openhab/userdata/etc/version.properties /openhab/userdata.dist/etc/version.properties
May 19 11:28:48 raspberrypi e404b082aef5[5425]: + [ ! -z ]
May 19 11:28:48 raspberrypi e404b082aef5[5425]: + ls -A /openhab/conf
May 19 11:28:48 raspberrypi e404b082aef5[5425]: + [ -z  ]
May 19 11:28:48 raspberrypi e404b082aef5[5425]: + echo No configuration found... initializing.
May 19 11:28:48 raspberrypi e404b082aef5[5425]: No configuration found... initializing.
May 19 11:28:48 raspberrypi e404b082aef5[5425]: + cp -av /openhab/conf.dist/. /openhab/conf/
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/conf.dist/./rules/readme.txt' -> '/openhab/conf/./rules/readme.txt'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/conf.dist/./rules' -> '/openhab/conf/./rules'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/conf.dist/./sounds/doorbell.mp3' -> '/openhab/conf/./sounds/doorbell.mp3'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/conf.dist/./sounds/barking.mp3' -> '/openhab/conf/./sounds/barking.mp3'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/conf.dist/./sounds' -> '/openhab/conf/./sounds'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/conf.dist/./scripts/readme.txt' -> '/openhab/conf/./scripts/readme.txt'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/conf.dist/./scripts' -> '/openhab/conf/./scripts'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/conf.dist/./items/readme.txt' -> '/openhab/conf/./items/readme.txt'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/conf.dist/./items' -> '/openhab/conf/./items'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/conf.dist/./things/readme.txt' -> '/openhab/conf/./things/readme.txt'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/conf.dist/./things' -> '/openhab/conf/./things'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/conf.dist/./html/readme.txt' -> '/openhab/conf/./html/readme.txt'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/conf.dist/./html/index.html' -> '/openhab/conf/./html/index.html'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/conf.dist/./html' -> '/openhab/conf/./html'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/conf.dist/./sitemaps/readme.txt' -> '/openhab/conf/./sitemaps/readme.txt'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/conf.dist/./sitemaps' -> '/openhab/conf/./sitemaps'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/conf.dist/./services/readme.txt' -> '/openhab/conf/./services/readme.txt'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/conf.dist/./services/addons.cfg' -> '/openhab/conf/./services/addons.cfg'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/conf.dist/./services/runtime.cfg' -> '/openhab/conf/./services/runtime.cfg'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/conf.dist/./services' -> '/openhab/conf/./services'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/conf.dist/./transform/en.map' -> '/openhab/conf/./transform/en.map'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/conf.dist/./transform/readme.txt' -> '/openhab/conf/./transform/readme.txt'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/conf.dist/./transform/de.map' -> '/openhab/conf/./transform/de.map'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/conf.dist/./transform' -> '/openhab/conf/./transform'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/conf.dist/./persistence/readme.txt' -> '/openhab/conf/./persistence/readme.txt'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/conf.dist/./persistence' -> '/openhab/conf/./persistence'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/conf.dist/./icons/classic/readme.txt' -> '/openhab/conf/./icons/classic/readme.txt'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/conf.dist/./icons/classic' -> '/openhab/conf/./icons/classic'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/conf.dist/./icons' -> '/openhab/conf/./icons'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: '/openhab/conf.dist/.' -> '/openhab/conf/.'
May 19 11:28:48 raspberrypi e404b082aef5[5425]: + chown -R openhab:openhab /openhab
May 19 11:29:21 raspberrypi e404b082aef5[5425]: + exec ./start_debug.sh
May 19 11:29:21 raspberrypi e404b082aef5[5425]: Launching the openHAB runtime...
May 19 11:29:23 raspberrypi kernel: [ 5093.963527] ------------[ cut here ]------------
May 19 11:29:23 raspberrypi kernel: [ 5093.963559] WARNING: CPU: 1 PID: 5885 at fs/overlayfs/readdir.c:393 ovl_iterate+0x2cc/0x2d4 [overlay]
May 19 11:29:23 raspberrypi kernel: [ 5093.963562] Modules linked in: ipt_MASQUERADE nf_nat_masquerade_ipv4 nf_conntrack_netlink nfnetlink iptable_nat nf_conntrack_ipv4 nf_defrag_ipv4 nf_nat_ipv4 xt_addrtype iptable_filter xt_conntrack nf_nat nf_conntrack br_netfilter bridge stp llc overlay cmac bnep hci_uart btbcm serdev bluetooth ecdh_generic brcmfmac brcmutil cfg80211 rfkill snd_bcm2835(C) snd_pcm snd_timer snd fixed uio_pdrv_genirq uio ip_tables x_tables ipv6
May 19 11:29:23 raspberrypi kernel: [ 5093.963659] CPU: 1 PID: 5885 Comm: karaf Tainted: G        WC      4.14.34-v7+ #1110
May 19 11:29:23 raspberrypi kernel: [ 5093.963661] Hardware name: BCM2835
May 19 11:29:23 raspberrypi kernel: [ 5093.963679] [<8010ffd8>] (unwind_backtrace) from [<8010c240>] (show_stack+0x20/0x24)
May 19 11:29:23 raspberrypi kernel: [ 5093.963690] [<8010c240>] (show_stack) from [<807840a4>] (dump_stack+0xd4/0x118)
May 19 11:29:23 raspberrypi kernel: [ 5093.963700] [<807840a4>] (dump_stack) from [<8011da54>] (__warn+0xf8/0x110)
May 19 11:29:23 raspberrypi kernel: [ 5093.963707] [<8011da54>] (__warn) from [<8011db3c>] (warn_slowpath_null+0x30/0x38)
May 19 11:29:23 raspberrypi kernel: [ 5093.963728] [<8011db3c>] (warn_slowpath_null) from [<7f46fe30>] (ovl_iterate+0x2cc/0x2d4 [overlay])
May 19 11:29:23 raspberrypi kernel: [ 5093.963752] [<7f46fe30>] (ovl_iterate [overlay]) from [<8029e494>] (iterate_dir+0x130/0x180)
May 19 11:29:23 raspberrypi kernel: [ 5093.963761] [<8029e494>] (iterate_dir) from [<8029e978>] (SyS_getdents64+0x88/0x174)
May 19 11:29:23 raspberrypi kernel: [ 5093.963769] [<8029e978>] (SyS_getdents64) from [<801081e0>] (__sys_trace_return+0x0/0x10)
May 19 11:29:23 raspberrypi kernel: [ 5093.963774] ---[ end trace 57212dc57ddf69b5 ]---
May 19 11:29:23 raspberrypi kernel: [ 5093.979438] ------------[ cut here ]------------
May 19 11:29:23 raspberrypi kernel: [ 5093.979474] WARNING: CPU: 3 PID: 6000 at fs/overlayfs/readdir.c:393 ovl_iterate+0x2cc/0x2d4 [overlay]
May 19 11:29:23 raspberrypi kernel: [ 5093.979477] Modules linked in: ipt_MASQUERADE nf_nat_masquerade_ipv4 nf_conntrack_netlink nfnetlink iptable_nat nf_conntrack_ipv4 nf_defrag_ipv4 nf_nat_ipv4 xt_addrtype iptable_filter xt_conntrack nf_nat nf_conntrack br_netfilter bridge stp llc overlay cmac bnep hci_uart btbcm serdev bluetooth ecdh_generic brcmfmac brcmutil cfg80211 rfkill snd_bcm2835(C) snd_pcm snd_timer snd fixed uio_pdrv_genirq uio ip_tables x_tables ipv6
May 19 11:29:23 raspberrypi kernel: [ 5093.979573] CPU: 3 PID: 6000 Comm: java Tainted: G        WC      4.14.34-v7+ #1110
May 19 11:29:23 raspberrypi kernel: [ 5093.979575] Hardware name: BCM2835
May 19 11:29:23 raspberrypi kernel: [ 5093.979594] [<8010ffd8>] (unwind_backtrace) from [<8010c240>] (show_stack+0x20/0x24)
May 19 11:29:23 raspberrypi kernel: [ 5093.979605] [<8010c240>] (show_stack) from [<807840a4>] (dump_stack+0xd4/0x118)
May 19 11:29:23 raspberrypi kernel: [ 5093.979615] [<807840a4>] (dump_stack) from [<8011da54>] (__warn+0xf8/0x110)
May 19 11:29:23 raspberrypi kernel: [ 5093.979622] [<8011da54>] (__warn) from [<8011db3c>] (warn_slowpath_null+0x30/0x38)
May 19 11:29:23 raspberrypi kernel: [ 5093.979643] [<8011db3c>] (warn_slowpath_null) from [<7f46fe30>] (ovl_iterate+0x2cc/0x2d4 [overlay])
May 19 11:29:23 raspberrypi kernel: [ 5093.979668] [<7f46fe30>] (ovl_iterate [overlay]) from [<8029e494>] (iterate_dir+0x130/0x180)
May 19 11:29:23 raspberrypi kernel: [ 5093.979676] [<8029e494>] (iterate_dir) from [<8029e978>] (SyS_getdents64+0x88/0x174)
May 19 11:29:23 raspberrypi kernel: [ 5093.979685] [<8029e978>] (SyS_getdents64) from [<801081e0>] (__sys_trace_return+0x0/0x10)
May 19 11:29:23 raspberrypi kernel: [ 5093.979689] ---[ end trace 57212dc57ddf69b6 ]---
May 19 11:29:23 raspberrypi kernel: [ 5094.331015] ------------[ cut here ]------------
May 19 11:29:23 raspberrypi kernel: [ 5094.331052] WARNING: CPU: 3 PID: 6000 at fs/overlayfs/readdir.c:393 ovl_iterate+0x2cc/0x2d4 [overlay]
May 19 11:29:23 raspberrypi kernel: [ 5094.331055] Modules linked in: ipt_MASQUERADE nf_nat_masquerade_ipv4 nf_conntrack_netlink nfnetlink iptable_nat nf_conntrack_ipv4 nf_defrag_ipv4 nf_nat_ipv4 xt_addrtype iptable_filter xt_conntrack nf_nat nf_conntrack br_netfilter bridge stp llc overlay cmac bnep hci_uart btbcm serdev bluetooth ecdh_generic brcmfmac brcmutil cfg80211 rfkill snd_bcm2835(C) snd_pcm snd_timer snd fixed uio_pdrv_genirq uio ip_tables x_tables ipv6
May 19 11:29:23 raspberrypi kernel: [ 5094.331151] CPU: 3 PID: 6000 Comm: java Tainted: G        WC      4.14.34-v7+ #1110
May 19 11:29:23 raspberrypi kernel: [ 5094.331154] Hardware name: BCM2835
May 19 11:29:23 raspberrypi kernel: [ 5094.331172] [<8010ffd8>] (unwind_backtrace) from [<8010c240>] (show_stack+0x20/0x24)
May 19 11:29:23 raspberrypi kernel: [ 5094.331183] [<8010c240>] (show_stack) from [<807840a4>] (dump_stack+0xd4/0x118)
May 19 11:29:23 raspberrypi kernel: [ 5094.331194] [<807840a4>] (dump_stack) from [<8011da54>] (__warn+0xf8/0x110)
May 19 11:29:23 raspberrypi kernel: [ 5094.331201] [<8011da54>] (__warn) from [<8011db3c>] (warn_slowpath_null+0x30/0x38)
May 19 11:29:23 raspberrypi kernel: [ 5094.331221] [<8011db3c>] (warn_slowpath_null) from [<7f46fe30>] (ovl_iterate+0x2cc/0x2d4 [overlay])
May 19 11:29:23 raspberrypi kernel: [ 5094.331245] [<7f46fe30>] (ovl_iterate [overlay]) from [<8029e494>] (iterate_dir+0x130/0x180)
May 19 11:29:23 raspberrypi kernel: [ 5094.331253] [<8029e494>] (iterate_dir) from [<8029e978>] (SyS_getdents64+0x88/0x174)
May 19 11:29:23 raspberrypi kernel: [ 5094.331262] [<8029e978>] (SyS_getdents64) from [<801081e0>] (__sys_trace_return+0x0/0x10)
May 19 11:29:23 raspberrypi kernel: [ 5094.331267] ---[ end trace 57212dc57ddf69b7 ]---
May 19 11:29:23 raspberrypi e404b082aef5[5425]: Listening for transport dt_socket at address: 5005
May 19 11:29:26 raspberrypi kernel: [ 5096.954395] ------------[ cut here ]------------
May 19 11:29:26 raspberrypi kernel: [ 5096.954431] WARNING: CPU: 3 PID: 6000 at fs/overlayfs/readdir.c:393 ovl_iterate+0x2cc/0x2d4 [overlay]
May 19 11:29:26 raspberrypi kernel: [ 5096.954434] Modules linked in: ipt_MASQUERADE nf_nat_masquerade_ipv4 nf_conntrack_netlink nfnetlink iptable_nat nf_conntrack_ipv4 nf_defrag_ipv4 nf_nat_ipv4 xt_addrtype iptable_filter xt_conntrack nf_nat nf_conntrack br_netfilter bridge stp llc overlay cmac bnep hci_uart btbcm serdev bluetooth ecdh_generic brcmfmac brcmutil cfg80211 rfkill snd_bcm2835(C) snd_pcm snd_timer snd fixed uio_pdrv_genirq uio ip_tables x_tables ipv6
May 19 11:29:26 raspberrypi kernel: [ 5096.954529] CPU: 3 PID: 6000 Comm: java Tainted: G        WC      4.14.34-v7+ #1110
May 19 11:29:26 raspberrypi kernel: [ 5096.954531] Hardware name: BCM2835
May 19 11:29:26 raspberrypi kernel: [ 5096.954549] [<8010ffd8>] (unwind_backtrace) from [<8010c240>] (show_stack+0x20/0x24)
May 19 11:29:26 raspberrypi kernel: [ 5096.954560] [<8010c240>] (show_stack) from [<807840a4>] (dump_stack+0xd4/0x118)
May 19 11:29:26 raspberrypi kernel: [ 5096.954570] [<807840a4>] (dump_stack) from [<8011da54>] (__warn+0xf8/0x110)
May 19 11:29:26 raspberrypi kernel: [ 5096.954578] [<8011da54>] (__warn) from [<8011db3c>] (warn_slowpath_null+0x30/0x38)
May 19 11:29:26 raspberrypi kernel: [ 5096.954598] [<8011db3c>] (warn_slowpath_null) from [<7f46fe30>] (ovl_iterate+0x2cc/0x2d4 [overlay])
May 19 11:29:26 raspberrypi kernel: [ 5096.954622] [<7f46fe30>] (ovl_iterate [overlay]) from [<8029e494>] (iterate_dir+0x130/0x180)
May 19 11:29:26 raspberrypi kernel: [ 5096.954630] [<8029e494>] (iterate_dir) from [<8029e978>] (SyS_getdents64+0x88/0x174)
May 19 11:29:26 raspberrypi kernel: [ 5096.954638] [<8029e978>] (SyS_getdents64) from [<801081e0>] (__sys_trace_return+0x0/0x10)
May 19 11:29:26 raspberrypi kernel: [ 5096.954643] ---[ end trace 57212dc57ddf69b8 ]---
May 19 11:31:53 raspberrypi e404b082aef5[5425]:
May 19 11:31:53 raspberrypi e404b082aef5[5425]: #033[31m                         #033[0m __  _____    ____      #033[0m
May 19 11:31:53 raspberrypi e404b082aef5[5425]: #033[31m  ____  ____  ___  ____  #033[0m/ / / /   |  / __ )     #033[0m
May 19 11:31:53 raspberrypi e404b082aef5[5425]: #033[31m / __ \/ __ \/ _ \/ __ \#033[0m/ /_/ / /| | / __  | #033[0m
May 19 11:31:53 raspberrypi e404b082aef5[5425]: #033[31m/ /_/ / /_/ /  __/ / / / #033[0m__  / ___ |/ /_/ /      #033[0m
May 19 11:31:53 raspberrypi e404b082aef5[5425]: #033[31m\____/ .___/\___/_/ /_/#033[0m_/ /_/_/  |_/_____/     #033[0m
May 19 11:31:53 raspberrypi e404b082aef5[5425]: #033[31m    /_/                       #033[0m 2.2.0#033[0m
May 19 11:31:53 raspberrypi e404b082aef5[5425]: #033[31m                              #033[0m Release Build   #033[0m
May 19 11:31:53 raspberrypi e404b082aef5[5425]:
May 19 11:31:53 raspberrypi e404b082aef5[5425]: Hit '#033[1m<tab>#033[0m' for a list of available commands
May 19 11:31:53 raspberrypi e404b082aef5[5425]: and '#033[1m[cmd] --help#033[0m' for help on a specific command.
May 19 11:31:53 raspberrypi e404b082aef5[5425]: Hit '#033[1m<ctrl-d>#033[0m' or type '#033[1msystem:shutdown#033[0m' or '#033[1mlogout#033[0m' to shutdown openHAB.
May 19 11:31:53 raspberrypi e404b082aef5[5425]:
May 19 11:32:23 raspberrypi e404b082aef5[5425]: #033[31mError executing command: java.io.IOException: Unable to parse columns#033[39m
May 19 11:32:26 raspberrypi dockerd[5425]: time="2018-05-19T11:32:26+02:00" level=info msg="shim reaped" id=e404b082aef5c4621883de2814ebc927d2221ec47236dd4023095c621bccdc74 module="containerd/tasks"
May 19 11:32:26 raspberrypi dockerd[5425]: time="2018-05-19T11:32:26.248516844+02:00" level=info msg="ignoring event" module=libcontainerd namespace=moby topic=/tasks/delete type="*events.TaskDelete"
```

In the second iteration there are no more kernel warnings but the image goes into an endless restart loop.
```
May 19 11:32:27 raspberrypi dockerd[5425]: time="2018-05-19T11:32:27+02:00" level=info msg="shim docker-containerd-shim started" address="/containerd-shim/moby/e404b082aef5c4621883de2814ebc927d2221ec47236dd4023095c621bccdc74/shim.sock" debug=false module="containerd/tasks" pid=6156
May 19 11:32:27 raspberrypi dockerd[5425]: time="2018-05-19T11:32:27+02:00" level=warning msg="OOM monitoring failed" error="cgroups: memory cgroup not supported on this system"
May 19 11:32:27 raspberrypi e404b082aef5[5425]: + test -t 0
May 19 11:32:27 raspberrypi e404b082aef5[5425]: + [ 0 -eq 1 ]
May 19 11:32:27 raspberrypi e404b082aef5[5425]: + set -euo pipefail
May 19 11:32:27 raspberrypi e404b082aef5[5425]: + IFS=
May 19 11:32:27 raspberrypi e404b082aef5[5425]:
May 19 11:32:27 raspberrypi e404b082aef5[5425]: + [ limited = unlimited ]
May 19 11:32:27 raspberrypi e404b082aef5[5425]: + rm -f /openhab/runtime/instances/instance.properties
May 19 11:32:27 raspberrypi e404b082aef5[5425]: + rm -f /openhab/userdata/tmp/instances/instance.properties
May 19 11:32:27 raspberrypi e404b082aef5[5425]: + NEW_USER_ID=9001
May 19 11:32:27 raspberrypi e404b082aef5[5425]: + echo Starting with openhab user id: 9001
May 19 11:32:27 raspberrypi e404b082aef5[5425]: Starting with openhab user id: 9001
May 19 11:32:27 raspberrypi e404b082aef5[5425]: + id -u openhab
May 19 11:32:27 raspberrypi e404b082aef5[5425]: + ls -A /openhab/userdata
May 19 11:32:27 raspberrypi e404b082aef5[5425]: + [ -z cache
May 19 11:32:27 raspberrypi e404b082aef5[5425]: config
May 19 11:32:27 raspberrypi e404b082aef5[5425]: etc
May 19 11:32:27 raspberrypi e404b082aef5[5425]: logs
May 19 11:32:27 raspberrypi e404b082aef5[5425]: tmp ]
May 19 11:32:27 raspberrypi e404b082aef5[5425]: + cmp /openhab/userdata/etc/version.properties /openhab/userdata.dist/etc/version.properties
May 19 11:32:27 raspberrypi e404b082aef5[5425]: + [ ! -z ]
May 19 11:32:27 raspberrypi e404b082aef5[5425]: + ls -A /openhab/conf
May 19 11:32:27 raspberrypi e404b082aef5[5425]: + [ -z html
May 19 11:32:27 raspberrypi e404b082aef5[5425]: icons
May 19 11:32:27 raspberrypi e404b082aef5[5425]: items
May 19 11:32:27 raspberrypi e404b082aef5[5425]: persistence
May 19 11:32:27 raspberrypi e404b082aef5[5425]: rules
May 19 11:32:27 raspberrypi e404b082aef5[5425]: scripts
May 19 11:32:27 raspberrypi e404b082aef5[5425]: services
May 19 11:32:27 raspberrypi e404b082aef5[5425]: sitemaps
May 19 11:32:27 raspberrypi e404b082aef5[5425]: sounds
May 19 11:32:27 raspberrypi e404b082aef5[5425]: things
May 19 11:32:27 raspberrypi e404b082aef5[5425]: transform ]
May 19 11:32:27 raspberrypi e404b082aef5[5425]: + chown -R openhab:openhab /openhab
May 19 11:32:28 raspberrypi e404b082aef5[5425]: + exec ./start_debug.sh
May 19 11:32:28 raspberrypi e404b082aef5[5425]: Launching the openHAB runtime...
May 19 11:32:29 raspberrypi e404b082aef5[5425]: Listening for transport dt_socket at address: 5005
May 19 11:34:11 raspberrypi e404b082aef5[5425]:
May 19 11:34:11 raspberrypi e404b082aef5[5425]: #033[31m                         #033[0m __  _____    ____      #033[0m
May 19 11:34:11 raspberrypi e404b082aef5[5425]: #033[31m  ____  ____  ___  ____  #033[0m/ / / /   |  / __ )     #033[0m
May 19 11:34:11 raspberrypi e404b082aef5[5425]: #033[31m / __ \/ __ \/ _ \/ __ \#033[0m/ /_/ / /| | / __  | #033[0m
May 19 11:34:11 raspberrypi e404b082aef5[5425]: #033[31m/ /_/ / /_/ /  __/ / / / #033[0m__  / ___ |/ /_/ /      #033[0m
May 19 11:34:11 raspberrypi e404b082aef5[5425]: #033[31m\____/ .___/\___/_/ /_/#033[0m_/ /_/_/  |_/_____/     #033[0m
May 19 11:34:11 raspberrypi e404b082aef5[5425]: #033[31m    /_/                       #033[0m 2.2.0#033[0m
May 19 11:34:11 raspberrypi e404b082aef5[5425]: #033[31m                              #033[0m Release Build   #033[0m
May 19 11:34:11 raspberrypi e404b082aef5[5425]:
May 19 11:34:11 raspberrypi e404b082aef5[5425]: Hit '#033[1m<tab>#033[0m' for a list of available commands
May 19 11:34:11 raspberrypi e404b082aef5[5425]: and '#033[1m[cmd] --help#033[0m' for help on a specific command.
May 19 11:34:11 raspberrypi e404b082aef5[5425]: Hit '#033[1m<ctrl-d>#033[0m' or type '#033[1msystem:shutdown#033[0m' or '#033[1mlogout#033[0m' to shutdown openHAB.
May 19 11:34:11 raspberrypi e404b082aef5[5425]:
May 19 11:34:40 raspberrypi e404b082aef5[5425]: #033[31mError executing command: java.io.IOException: Unable to parse columns#033[39m
May 19 11:34:43 raspberrypi dockerd[5425]: time="2018-05-19T11:34:43+02:00" level=info msg="shim reaped" id=e404b082aef5c4621883de2814ebc927d2221ec47236dd4023095c621bccdc74 module="containerd/tasks"
May 19 11:34:43 raspberrypi dockerd[5425]: time="2018-05-19T11:34:43.370290990+02:00" level=info msg="ignoring event" module=libcontainerd namespace=moby topic=/tasks/delete type="*events.TaskDelete"
May 19 11:34:44 raspberrypi dockerd[5425]: time="2018-05-19T11:34:44+02:00" level=info msg="shim docker-containerd-shim started" address="/containerd-shim/moby/e404b082aef5c4621883de2814ebc927d2221ec47236dd4023095c621bccdc74/shim.sock" debug=false module="containerd/tasks" pid=6411
May 19 11:34:44 raspberrypi dockerd[5425]: time="2018-05-19T11:34:44+02:00" level=warning msg="OOM monitoring failed" error="cgroups: memory cgroup not supported on this system"
May 19 11:34:44 raspberrypi e404b082aef5[5425]: + test -t 0
May 19 11:34:44 raspberrypi e404b082aef5[5425]: + [ 0 -eq 1 ]
May 19 11:34:44 raspberrypi e404b082aef5[5425]: + set -euo pipefail
May 19 11:34:44 raspberrypi e404b082aef5[5425]: + IFS=
May 19 11:34:44 raspberrypi e404b082aef5[5425]:
May 19 11:34:44 raspberrypi e404b082aef5[5425]: + [ limited = unlimited ]
May 19 11:34:44 raspberrypi e404b082aef5[5425]: + rm -f /openhab/runtime/instances/instance.properties
May 19 11:34:44 raspberrypi e404b082aef5[5425]: + rm -f /openhab/userdata/tmp/instances/instance.properties
May 19 11:34:44 raspberrypi e404b082aef5[5425]: + NEW_USER_ID=9001
May 19 11:34:44 raspberrypi e404b082aef5[5425]: + echo Starting with openhab user id: 9001
May 19 11:34:44 raspberrypi e404b082aef5[5425]: Starting with openhab user id: 9001
May 19 11:34:44 raspberrypi e404b082aef5[5425]: + id -u openhab
May 19 11:34:44 raspberrypi e404b082aef5[5425]: + ls -A /openhab/userdata
May 19 11:34:44 raspberrypi e404b082aef5[5425]: + [ -z cache
May 19 11:34:44 raspberrypi e404b082aef5[5425]: config
May 19 11:34:44 raspberrypi e404b082aef5[5425]: etc
May 19 11:34:44 raspberrypi e404b082aef5[5425]: logs
May 19 11:34:44 raspberrypi e404b082aef5[5425]: tmp ]
May 19 11:34:44 raspberrypi e404b082aef5[5425]: + cmp /openhab/userdata/etc/version.properties /openhab/userdata.dist/etc/version.properties
May 19 11:34:44 raspberrypi e404b082aef5[5425]: + [ ! -z ]
May 19 11:34:44 raspberrypi e404b082aef5[5425]: + ls -A /openhab/conf
May 19 11:34:44 raspberrypi e404b082aef5[5425]: + [ -z html
May 19 11:34:44 raspberrypi e404b082aef5[5425]: icons
May 19 11:34:44 raspberrypi e404b082aef5[5425]: items
May 19 11:34:44 raspberrypi e404b082aef5[5425]: persistence
May 19 11:34:44 raspberrypi e404b082aef5[5425]: rules
May 19 11:34:44 raspberrypi e404b082aef5[5425]: scripts
May 19 11:34:44 raspberrypi e404b082aef5[5425]: services
May 19 11:34:44 raspberrypi e404b082aef5[5425]: sitemaps
May 19 11:34:44 raspberrypi e404b082aef5[5425]: sounds
May 19 11:34:44 raspberrypi e404b082aef5[5425]: things
May 19 11:34:44 raspberrypi e404b082aef5[5425]: transform ]
May 19 11:34:44 raspberrypi e404b082aef5[5425]: + chown -R openhab:openhab /openhab
May 19 11:34:44 raspberrypi e404b082aef5[5425]: + exec ./start_debug.sh
May 19 11:34:44 raspberrypi e404b082aef5[5425]: Launching the openHAB runtime...
May 19 11:34:46 raspberrypi e404b082aef5[5425]: Listening for transport dt_socket at address: 5005
May 19 11:36:27 raspberrypi e404b082aef5[5425]:
May 19 11:36:27 raspberrypi e404b082aef5[5425]: #033[31m                         #033[0m __  _____    ____      #033[0m
May 19 11:36:27 raspberrypi e404b082aef5[5425]: #033[31m  ____  ____  ___  ____  #033[0m/ / / /   |  / __ )     #033[0m
May 19 11:36:27 raspberrypi e404b082aef5[5425]: #033[31m / __ \/ __ \/ _ \/ __ \#033[0m/ /_/ / /| | / __  | #033[0m
May 19 11:36:27 raspberrypi e404b082aef5[5425]: #033[31m/ /_/ / /_/ /  __/ / / / #033[0m__  / ___ |/ /_/ /      #033[0m
May 19 11:36:27 raspberrypi e404b082aef5[5425]: #033[31m\____/ .___/\___/_/ /_/#033[0m_/ /_/_/  |_/_____/     #033[0m
May 19 11:36:27 raspberrypi e404b082aef5[5425]: #033[31m    /_/                       #033[0m 2.2.0#033[0m
May 19 11:36:27 raspberrypi e404b082aef5[5425]: #033[31m                              #033[0m Release Build   #033[0m
May 19 11:36:27 raspberrypi e404b082aef5[5425]:
May 19 11:36:27 raspberrypi e404b082aef5[5425]: Hit '#033[1m<tab>#033[0m' for a list of available commands
May 19 11:36:27 raspberrypi e404b082aef5[5425]: and '#033[1m[cmd] --help#033[0m' for help on a specific command.
May 19 11:36:27 raspberrypi e404b082aef5[5425]: Hit '#033[1m<ctrl-d>#033[0m' or type '#033[1msystem:shutdown#033[0m' or '#033[1mlogout#033[0m' to shutdown openHAB.
May 19 11:36:27 raspberrypi e404b082aef5[5425]:
May 19 11:36:56 raspberrypi e404b082aef5[5425]: #033[31mError executing command: java.io.IOException: Unable to parse columns#033[39m
May 19 11:36:58 raspberrypi e404b082aef5[5425]: WARNING: EventAdmin: Exception: java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.FutureTask@3688a rejected from java.util.concurrent.ThreadPoolExecutor@c485fa[Terminated, pool size = 0, active threads = 0, queued tasks = 0, completed tasks = 133] (java.util.concurrent.RejectedExecutionException: Task java.util.concurrent.FutureTask@3688a rejected from java.util.concurrent.ThreadPoolExecutor@c485fa[Terminated, pool size = 0, active threads = 0, queued tasks = 0, completed tasks = 133])
May 19 11:36:59 raspberrypi dockerd[5425]: time="2018-05-19T11:36:59+02:00" level=info msg="shim reaped" id=e404b082aef5c4621883de2814ebc927d2221ec47236dd4023095c621bccdc74 module="containerd/tasks"
May 19 11:36:59 raspberrypi dockerd[5425]: time="2018-05-19T11:36:59.405274025+02:00" level=info msg="ignoring event" module=libcontainerd namespace=moby topic=/tasks/delete type="*events.TaskDelete"
```

For completeness the root cause seems to be that the jansi library fails to load on armhf as @zarusz already posted in the [community topic](https://community.openhab.org/t/docker-openhab-2-2-0-and-2-3-0-snapshot-arm64-ioexception-unable-to-parse-columns/36888/4?u=wborn).

When increasing the log level in `etc/org.ops4j.pax.logging.cfg` to `log4j2.rootLogger.level = DEBUG` the  following exceptions are logged:

```
2018-06-16 17:36:13.003 [DEBUG] [org.jline                           ] - Error creating JNA based terminal: com/sun/jna/Platform
java.lang.NoClassDefFoundError: com/sun/jna/Platform
	at org.jline.terminal.impl.jna.JnaNativePty.current(JnaNativePty.java:40) ~[14:org.jline:3.6.0]
	at org.jline.terminal.impl.jna.JnaSupportImpl.current(JnaSupportImpl.java:16) ~[14:org.jline:3.6.0]
	at org.jline.terminal.TerminalBuilder.doBuild(TerminalBuilder.java:320) [14:org.jline:3.6.0]
	at org.jline.terminal.TerminalBuilder.build(TerminalBuilder.java:222) [14:org.jline:3.6.0]
	at org.apache.karaf.shell.impl.console.osgi.LocalConsoleManager.start(LocalConsoleManager.java:65) [12:org.apache.karaf.shell.core:4.1.5]
	at org.apache.karaf.shell.impl.console.osgi.Activator.start(Activator.java:114) [12:org.apache.karaf.shell.core:4.1.5]
	at org.eclipse.osgi.internal.framework.BundleContextImpl$3.run(BundleContextImpl.java:779) [?:?]
	at org.eclipse.osgi.internal.framework.BundleContextImpl$3.run(BundleContextImpl.java:1) [?:?]
	at java.security.AccessController.doPrivileged(Native Method) ~[?:?]
	at org.eclipse.osgi.internal.framework.BundleContextImpl.startActivator(BundleContextImpl.java:772) [?:?]
	at org.eclipse.osgi.internal.framework.BundleContextImpl.start(BundleContextImpl.java:729) [?:?]
	at org.eclipse.osgi.internal.framework.EquinoxBundle.startWorker0(EquinoxBundle.java:933) [?:?]
	at org.eclipse.osgi.internal.framework.EquinoxBundle$EquinoxModule.startWorker(EquinoxBundle.java:309) [?:?]
	at org.eclipse.osgi.container.Module.doStart(Module.java:581) [?:?]
	at org.eclipse.osgi.container.Module.start(Module.java:449) [?:?]
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1634) [?:?]
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1614) [?:?]
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1585) [?:?]
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1528) [?:?]
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1) [?:?]
	at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:230) [?:?]
	at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:340) [?:?]
Caused by: java.lang.ClassNotFoundException: com.sun.jna.Platform cannot be found by org.jline_3.6.0
	at org.eclipse.osgi.internal.loader.BundleLoader.findClassInternal(BundleLoader.java:484) ~[?:?]
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:395) ~[?:?]
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:387) ~[?:?]
	at org.eclipse.osgi.internal.loader.ModuleClassLoader.loadClass(ModuleClassLoader.java:150) ~[?:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357) ~[?:?]
	... 22 more
2018-06-16 17:36:13.493 [DEBUG] [org.jline                           ] - Error creating JANSI based terminal: Could not load library. Reasons: [java.lang.LinkageError: Unable to load library jansi32-1.17, java.lang.LinkageError: Unable to load library jansi-1.17, java.lang.LinkageError: Unable to load library jansi, java.lang.LinkageError: Unable to load library from /openhab/userdata/tmp/libjansi-1.17-7122820824891838856.so, java.lang.LinkageError: Unable to load library from /openhab/.hawtjni/jansi/libjansi-1.17-7219676414123987796.so]
java.lang.UnsatisfiedLinkError: Could not load library. Reasons: [java.lang.LinkageError: Unable to load library jansi32-1.17, java.lang.LinkageError: Unable to load library jansi-1.17, java.lang.LinkageError: Unable to load library jansi, java.lang.LinkageError: Unable to load library from /openhab/userdata/tmp/libjansi-1.17-7122820824891838856.so, java.lang.LinkageError: Unable to load library from /openhab/.hawtjni/jansi/libjansi-1.17-7219676414123987796.so]
	at org.fusesource.hawtjni.runtime.Library.doLoad(Library.java:224) ~[5:org.fusesource.jansi:1.17.0]
	at org.fusesource.hawtjni.runtime.Library.load(Library.java:176) ~[5:org.fusesource.jansi:1.17.0]
	at org.fusesource.jansi.internal.CLibrary.<clinit>(CLibrary.java:41) ~[5:org.fusesource.jansi:1.17.0]
	at org.jline.terminal.impl.jansi.JansiNativePty.ttyname(JansiNativePty.java:58) ~[14:org.jline:3.6.0]
	at org.jline.terminal.impl.jansi.linux.LinuxNativePty.current(LinuxNativePty.java:26) ~[14:org.jline:3.6.0]
	at org.jline.terminal.impl.jansi.JansiSupportImpl.current(JansiSupportImpl.java:54) ~[14:org.jline:3.6.0]
	at org.jline.terminal.TerminalBuilder.doBuild(TerminalBuilder.java:330) [14:org.jline:3.6.0]
	at org.jline.terminal.TerminalBuilder.build(TerminalBuilder.java:222) [14:org.jline:3.6.0]
	at org.apache.karaf.shell.impl.console.osgi.LocalConsoleManager.start(LocalConsoleManager.java:65) [12:org.apache.karaf.shell.core:4.1.5]
	at org.apache.karaf.shell.impl.console.osgi.Activator.start(Activator.java:114) [12:org.apache.karaf.shell.core:4.1.5]
	at org.eclipse.osgi.internal.framework.BundleContextImpl$3.run(BundleContextImpl.java:779) [?:?]
	at org.eclipse.osgi.internal.framework.BundleContextImpl$3.run(BundleContextImpl.java:1) [?:?]
	at java.security.AccessController.doPrivileged(Native Method) ~[?:?]
	at org.eclipse.osgi.internal.framework.BundleContextImpl.startActivator(BundleContextImpl.java:772) [?:?]
	at org.eclipse.osgi.internal.framework.BundleContextImpl.start(BundleContextImpl.java:729) [?:?]
	at org.eclipse.osgi.internal.framework.EquinoxBundle.startWorker0(EquinoxBundle.java:933) [?:?]
	at org.eclipse.osgi.internal.framework.EquinoxBundle$EquinoxModule.startWorker(EquinoxBundle.java:309) [?:?]
	at org.eclipse.osgi.container.Module.doStart(Module.java:581) [?:?]
	at org.eclipse.osgi.container.Module.start(Module.java:449) [?:?]
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1634) [?:?]
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1614) [?:?]
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1585) [?:?]
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1528) [?:?]
	at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1) [?:?]
	at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:230) [?:?]
	at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:340) [?:?]
	Suppressed: java.lang.LinkageError: Unable to load library jansi32-1.17
		at org.fusesource.hawtjni.runtime.Library.loadLibrary(Library.java:450) ~[5:org.fusesource.jansi:1.17.0]
		at org.fusesource.hawtjni.runtime.Library.doLoad(Library.java:204) ~[5:org.fusesource.jansi:1.17.0]
		at org.fusesource.hawtjni.runtime.Library.load(Library.java:176) ~[5:org.fusesource.jansi:1.17.0]
		at org.fusesource.jansi.internal.CLibrary.<clinit>(CLibrary.java:41) ~[5:org.fusesource.jansi:1.17.0]
		at org.jline.terminal.impl.jansi.JansiNativePty.ttyname(JansiNativePty.java:58) ~[14:org.jline:3.6.0]
		at org.jline.terminal.impl.jansi.linux.LinuxNativePty.current(LinuxNativePty.java:26) ~[14:org.jline:3.6.0]
		at org.jline.terminal.impl.jansi.JansiSupportImpl.current(JansiSupportImpl.java:54) ~[14:org.jline:3.6.0]
		at org.jline.terminal.TerminalBuilder.doBuild(TerminalBuilder.java:330) [14:org.jline:3.6.0]
		at org.jline.terminal.TerminalBuilder.build(TerminalBuilder.java:222) [14:org.jline:3.6.0]
		at org.apache.karaf.shell.impl.console.osgi.LocalConsoleManager.start(LocalConsoleManager.java:65) [12:org.apache.karaf.shell.core:4.1.5]
		at org.apache.karaf.shell.impl.console.osgi.Activator.start(Activator.java:114) [12:org.apache.karaf.shell.core:4.1.5]
		at org.eclipse.osgi.internal.framework.BundleContextImpl$3.run(BundleContextImpl.java:779) [?:?]
		at org.eclipse.osgi.internal.framework.BundleContextImpl$3.run(BundleContextImpl.java:1) [?:?]
		at java.security.AccessController.doPrivileged(Native Method) ~[?:?]
		at org.eclipse.osgi.internal.framework.BundleContextImpl.startActivator(BundleContextImpl.java:772) [?:?]
		at org.eclipse.osgi.internal.framework.BundleContextImpl.start(BundleContextImpl.java:729) [?:?]
		at org.eclipse.osgi.internal.framework.EquinoxBundle.startWorker0(EquinoxBundle.java:933) [?:?]
		at org.eclipse.osgi.internal.framework.EquinoxBundle$EquinoxModule.startWorker(EquinoxBundle.java:309) [?:?]
		at org.eclipse.osgi.container.Module.doStart(Module.java:581) [?:?]
		at org.eclipse.osgi.container.Module.start(Module.java:449) [?:?]
		at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1634) [?:?]
		at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1614) [?:?]
		at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1585) [?:?]
		at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1528) [?:?]
		at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1) [?:?]
		at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:230) [?:?]
		at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:340) [?:?]
	Caused by: java.lang.UnsatisfiedLinkError: no jansi32-1.17 in java.library.path
		at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1867) ~[?:?]
		at java.lang.Runtime.loadLibrary0(Runtime.java:870) ~[?:?]
		at java.lang.System.loadLibrary(System.java:1122) ~[?:?]
		at org.fusesource.hawtjni.runtime.Library.loadLibrary(Library.java:446) ~[?:?]
		... 26 more
	Suppressed: java.lang.LinkageError: Unable to load library jansi-1.17
		at org.fusesource.hawtjni.runtime.Library.loadLibrary(Library.java:450) ~[5:org.fusesource.jansi:1.17.0]
		at org.fusesource.hawtjni.runtime.Library.doLoad(Library.java:206) ~[5:org.fusesource.jansi:1.17.0]
		at org.fusesource.hawtjni.runtime.Library.load(Library.java:176) ~[5:org.fusesource.jansi:1.17.0]
		at org.fusesource.jansi.internal.CLibrary.<clinit>(CLibrary.java:41) ~[5:org.fusesource.jansi:1.17.0]
		at org.jline.terminal.impl.jansi.JansiNativePty.ttyname(JansiNativePty.java:58) ~[14:org.jline:3.6.0]
		at org.jline.terminal.impl.jansi.linux.LinuxNativePty.current(LinuxNativePty.java:26) ~[14:org.jline:3.6.0]
		at org.jline.terminal.impl.jansi.JansiSupportImpl.current(JansiSupportImpl.java:54) ~[14:org.jline:3.6.0]
		at org.jline.terminal.TerminalBuilder.doBuild(TerminalBuilder.java:330) [14:org.jline:3.6.0]
		at org.jline.terminal.TerminalBuilder.build(TerminalBuilder.java:222) [14:org.jline:3.6.0]
		at org.apache.karaf.shell.impl.console.osgi.LocalConsoleManager.start(LocalConsoleManager.java:65) [12:org.apache.karaf.shell.core:4.1.5]
		at org.apache.karaf.shell.impl.console.osgi.Activator.start(Activator.java:114) [12:org.apache.karaf.shell.core:4.1.5]
		at org.eclipse.osgi.internal.framework.BundleContextImpl$3.run(BundleContextImpl.java:779) [?:?]
		at org.eclipse.osgi.internal.framework.BundleContextImpl$3.run(BundleContextImpl.java:1) [?:?]
		at java.security.AccessController.doPrivileged(Native Method) ~[?:?]
		at org.eclipse.osgi.internal.framework.BundleContextImpl.startActivator(BundleContextImpl.java:772) [?:?]
		at org.eclipse.osgi.internal.framework.BundleContextImpl.start(BundleContextImpl.java:729) [?:?]
		at org.eclipse.osgi.internal.framework.EquinoxBundle.startWorker0(EquinoxBundle.java:933) [?:?]
		at org.eclipse.osgi.internal.framework.EquinoxBundle$EquinoxModule.startWorker(EquinoxBundle.java:309) [?:?]
		at org.eclipse.osgi.container.Module.doStart(Module.java:581) [?:?]
		at org.eclipse.osgi.container.Module.start(Module.java:449) [?:?]
		at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1634) [?:?]
		at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1614) [?:?]
		at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1585) [?:?]
		at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1528) [?:?]
		at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1) [?:?]
		at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:230) [?:?]
		at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:340) [?:?]
	Caused by: java.lang.UnsatisfiedLinkError: no jansi-1.17 in java.library.path
		at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1867) ~[?:?]
		at java.lang.Runtime.loadLibrary0(Runtime.java:870) ~[?:?]
		at java.lang.System.loadLibrary(System.java:1122) ~[?:?]
		at org.fusesource.hawtjni.runtime.Library.loadLibrary(Library.java:446) ~[?:?]
		... 26 more
	Suppressed: java.lang.LinkageError: Unable to load library jansi
		at org.fusesource.hawtjni.runtime.Library.loadLibrary(Library.java:450) ~[5:org.fusesource.jansi:1.17.0]
		at org.fusesource.hawtjni.runtime.Library.doLoad(Library.java:208) ~[5:org.fusesource.jansi:1.17.0]
		at org.fusesource.hawtjni.runtime.Library.load(Library.java:176) ~[5:org.fusesource.jansi:1.17.0]
		at org.fusesource.jansi.internal.CLibrary.<clinit>(CLibrary.java:41) ~[5:org.fusesource.jansi:1.17.0]
		at org.jline.terminal.impl.jansi.JansiNativePty.ttyname(JansiNativePty.java:58) ~[14:org.jline:3.6.0]
		at org.jline.terminal.impl.jansi.linux.LinuxNativePty.current(LinuxNativePty.java:26) ~[14:org.jline:3.6.0]
		at org.jline.terminal.impl.jansi.JansiSupportImpl.current(JansiSupportImpl.java:54) ~[14:org.jline:3.6.0]
		at org.jline.terminal.TerminalBuilder.doBuild(TerminalBuilder.java:330) [14:org.jline:3.6.0]
		at org.jline.terminal.TerminalBuilder.build(TerminalBuilder.java:222) [14:org.jline:3.6.0]
		at org.apache.karaf.shell.impl.console.osgi.LocalConsoleManager.start(LocalConsoleManager.java:65) [12:org.apache.karaf.shell.core:4.1.5]
		at org.apache.karaf.shell.impl.console.osgi.Activator.start(Activator.java:114) [12:org.apache.karaf.shell.core:4.1.5]
		at org.eclipse.osgi.internal.framework.BundleContextImpl$3.run(BundleContextImpl.java:779) [?:?]
		at org.eclipse.osgi.internal.framework.BundleContextImpl$3.run(BundleContextImpl.java:1) [?:?]
		at java.security.AccessController.doPrivileged(Native Method) ~[?:?]
		at org.eclipse.osgi.internal.framework.BundleContextImpl.startActivator(BundleContextImpl.java:772) [?:?]
		at org.eclipse.osgi.internal.framework.BundleContextImpl.start(BundleContextImpl.java:729) [?:?]
		at org.eclipse.osgi.internal.framework.EquinoxBundle.startWorker0(EquinoxBundle.java:933) [?:?]
		at org.eclipse.osgi.internal.framework.EquinoxBundle$EquinoxModule.startWorker(EquinoxBundle.java:309) [?:?]
		at org.eclipse.osgi.container.Module.doStart(Module.java:581) [?:?]
		at org.eclipse.osgi.container.Module.start(Module.java:449) [?:?]
		at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1634) [?:?]
		at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1614) [?:?]
		at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1585) [?:?]
		at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1528) [?:?]
		at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1) [?:?]
		at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:230) [?:?]
		at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:340) [?:?]
	Caused by: java.lang.UnsatisfiedLinkError: no jansi in java.library.path
		at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1867) ~[?:?]
		at java.lang.Runtime.loadLibrary0(Runtime.java:870) ~[?:?]
		at java.lang.System.loadLibrary(System.java:1122) ~[?:?]
		at org.fusesource.hawtjni.runtime.Library.loadLibrary(Library.java:446) ~[?:?]
		... 26 more
	Suppressed: java.lang.LinkageError: Unable to load library from /openhab/userdata/tmp/libjansi-1.17-7122820824891838856.so
		at org.fusesource.hawtjni.runtime.Library.load(Library.java:437) ~[5:org.fusesource.jansi:1.17.0]
		at org.fusesource.hawtjni.runtime.Library.extractAndLoad(Library.java:315) ~[5:org.fusesource.jansi:1.17.0]
		at org.fusesource.hawtjni.runtime.Library.doLoad(Library.java:218) ~[5:org.fusesource.jansi:1.17.0]
		at org.fusesource.hawtjni.runtime.Library.load(Library.java:176) ~[5:org.fusesource.jansi:1.17.0]
		at org.fusesource.jansi.internal.CLibrary.<clinit>(CLibrary.java:41) ~[5:org.fusesource.jansi:1.17.0]
		at org.jline.terminal.impl.jansi.JansiNativePty.ttyname(JansiNativePty.java:58) ~[14:org.jline:3.6.0]
		at org.jline.terminal.impl.jansi.linux.LinuxNativePty.current(LinuxNativePty.java:26) ~[14:org.jline:3.6.0]
		at org.jline.terminal.impl.jansi.JansiSupportImpl.current(JansiSupportImpl.java:54) ~[14:org.jline:3.6.0]
		at org.jline.terminal.TerminalBuilder.doBuild(TerminalBuilder.java:330) [14:org.jline:3.6.0]
		at org.jline.terminal.TerminalBuilder.build(TerminalBuilder.java:222) [14:org.jline:3.6.0]
		at org.apache.karaf.shell.impl.console.osgi.LocalConsoleManager.start(LocalConsoleManager.java:65) [12:org.apache.karaf.shell.core:4.1.5]
		at org.apache.karaf.shell.impl.console.osgi.Activator.start(Activator.java:114) [12:org.apache.karaf.shell.core:4.1.5]
		at org.eclipse.osgi.internal.framework.BundleContextImpl$3.run(BundleContextImpl.java:779) [?:?]
		at org.eclipse.osgi.internal.framework.BundleContextImpl$3.run(BundleContextImpl.java:1) [?:?]
		at java.security.AccessController.doPrivileged(Native Method) ~[?:?]
		at org.eclipse.osgi.internal.framework.BundleContextImpl.startActivator(BundleContextImpl.java:772) [?:?]
		at org.eclipse.osgi.internal.framework.BundleContextImpl.start(BundleContextImpl.java:729) [?:?]
		at org.eclipse.osgi.internal.framework.EquinoxBundle.startWorker0(EquinoxBundle.java:933) [?:?]
		at org.eclipse.osgi.internal.framework.EquinoxBundle$EquinoxModule.startWorker(EquinoxBundle.java:309) [?:?]
		at org.eclipse.osgi.container.Module.doStart(Module.java:581) [?:?]
		at org.eclipse.osgi.container.Module.start(Module.java:449) [?:?]
		at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1634) [?:?]
		at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1614) [?:?]
		at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1585) [?:?]
		at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1528) [?:?]
		at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1) [?:?]
		at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:230) [?:?]
		at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:340) [?:?]
	Caused by: java.lang.UnsatisfiedLinkError: /openhab/userdata/tmp/libjansi-1.17-7122820824891838856.so: Error relocating /openhab/userdata/tmp/libjansi-1.17-7122820824891838856.so: unsupported relocation type 7 (Possible cause: can't load IA 32-bit .so on a ARM-bit platform)
		at java.lang.ClassLoader$NativeLibrary.load(Native Method) ~[?:?]
		at java.lang.ClassLoader.loadLibrary0(ClassLoader.java:1941) ~[?:?]
		at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1824) ~[?:?]
		at java.lang.Runtime.load0(Runtime.java:809) ~[?:?]
		at java.lang.System.load(System.java:1086) ~[?:?]
		at org.fusesource.hawtjni.runtime.Library.load(Library.java:433) ~[?:?]
		... 27 more
	Suppressed: java.lang.LinkageError: Unable to load library from /openhab/.hawtjni/jansi/libjansi-1.17-7219676414123987796.so
		at org.fusesource.hawtjni.runtime.Library.load(Library.java:437) ~[5:org.fusesource.jansi:1.17.0]
		at org.fusesource.hawtjni.runtime.Library.extractAndLoad(Library.java:315) ~[5:org.fusesource.jansi:1.17.0]
		at org.fusesource.hawtjni.runtime.Library.doLoad(Library.java:218) ~[5:org.fusesource.jansi:1.17.0]
		at org.fusesource.hawtjni.runtime.Library.load(Library.java:176) ~[5:org.fusesource.jansi:1.17.0]
		at org.fusesource.jansi.internal.CLibrary.<clinit>(CLibrary.java:41) ~[5:org.fusesource.jansi:1.17.0]
		at org.jline.terminal.impl.jansi.JansiNativePty.ttyname(JansiNativePty.java:58) ~[14:org.jline:3.6.0]
		at org.jline.terminal.impl.jansi.linux.LinuxNativePty.current(LinuxNativePty.java:26) ~[14:org.jline:3.6.0]
		at org.jline.terminal.impl.jansi.JansiSupportImpl.current(JansiSupportImpl.java:54) ~[14:org.jline:3.6.0]
		at org.jline.terminal.TerminalBuilder.doBuild(TerminalBuilder.java:330) [14:org.jline:3.6.0]
		at org.jline.terminal.TerminalBuilder.build(TerminalBuilder.java:222) [14:org.jline:3.6.0]
		at org.apache.karaf.shell.impl.console.osgi.LocalConsoleManager.start(LocalConsoleManager.java:65) [12:org.apache.karaf.shell.core:4.1.5]
		at org.apache.karaf.shell.impl.console.osgi.Activator.start(Activator.java:114) [12:org.apache.karaf.shell.core:4.1.5]
		at org.eclipse.osgi.internal.framework.BundleContextImpl$3.run(BundleContextImpl.java:779) [?:?]
		at org.eclipse.osgi.internal.framework.BundleContextImpl$3.run(BundleContextImpl.java:1) [?:?]
		at java.security.AccessController.doPrivileged(Native Method) ~[?:?]
		at org.eclipse.osgi.internal.framework.BundleContextImpl.startActivator(BundleContextImpl.java:772) [?:?]
		at org.eclipse.osgi.internal.framework.BundleContextImpl.start(BundleContextImpl.java:729) [?:?]
		at org.eclipse.osgi.internal.framework.EquinoxBundle.startWorker0(EquinoxBundle.java:933) [?:?]
		at org.eclipse.osgi.internal.framework.EquinoxBundle$EquinoxModule.startWorker(EquinoxBundle.java:309) [?:?]
		at org.eclipse.osgi.container.Module.doStart(Module.java:581) [?:?]
		at org.eclipse.osgi.container.Module.start(Module.java:449) [?:?]
		at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1634) [?:?]
		at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.incStartLevel(ModuleContainer.java:1614) [?:?]
		at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.doContainerStartLevel(ModuleContainer.java:1585) [?:?]
		at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1528) [?:?]
		at org.eclipse.osgi.container.ModuleContainer$ContainerStartLevel.dispatchEvent(ModuleContainer.java:1) [?:?]
		at org.eclipse.osgi.framework.eventmgr.EventManager.dispatchEvent(EventManager.java:230) [?:?]
		at org.eclipse.osgi.framework.eventmgr.EventManager$EventThread.run(EventManager.java:340) [?:?]
	Caused by: java.lang.UnsatisfiedLinkError: /openhab/.hawtjni/jansi/libjansi-1.17-7219676414123987796.so: Error relocating /openhab/.hawtjni/jansi/libjansi-1.17-7219676414123987796.so: unsupported relocation type 7 (Possible cause: can't load IA 32-bit .so on a ARM-bit platform)
		at java.lang.ClassLoader$NativeLibrary.load(Native Method) ~[?:?]
		at java.lang.ClassLoader.loadLibrary0(ClassLoader.java:1941) ~[?:?]
		at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1824) ~[?:?]
		at java.lang.Runtime.load0(Runtime.java:809) ~[?:?]
		at java.lang.System.load(System.java:1086) ~[?:?]
		at org.fusesource.hawtjni.runtime.Library.load(Library.java:433) ~[?:?]
		... 27 more
```

--- a/alpine/Dockerfile
+++ b/alpine/Dockerfile
@@ -46,7 +46,9 @@ RUN apk update --no-cache && \
         bash \
         ca-certificates \
         curl \
+        eudev \
         fontconfig \
+        gcompat \
         libcap \
         nss \
         shadow \

--- a/alpine/entrypoint
+++ b/alpine/entrypoint
@@ -37,8 +37,10 @@ if ! id -u openhab >/dev/null 2>&1; then
   adduser -u $NEW_USER_ID -D -g '' -h ${OPENHAB_HOME} -G openhab openhab
   adduser openhab dialout
   adduser openhab uucp
-fi
 
+  chown root:uucp /var/lock
+  chmod g+w /var/lock
+fi
 
 initialize_volume() {
   volume="$1"

